network/benchmarks/netperf/launch.go

```diff
@@ -74,7 +74,7 @@ func init() {
 	flag.IntVar(&iterations, "iterations", 1,
 		"Number of iterations to run")
 	flag.StringVar(&tag, "tag", runUUID, "CSV file suffix")
-	flag.StringVar(&netperfImage, "image", "sirot/netperf-latest", "Docker image used to run the network tests")
+	flag.StringVar(&netperfImage, "image", "gcr.io/k8s-staging-perf-tests/netperf", "Docker image used to run the network tests")
```

Comment:
```suggestion
	flag.StringVar(&netperfImage, "image", "gcr.io/k8s-staging-perf-tests/netperf:v0.0.1", "Docker image used to run the network tests")
```

It seems this image is tagged

Reply:
hmm, I've added tagging image as `latest` too, so we don't need to specify tag here/update it each time we push new image.

Reply:
I see, confused the code changes

Reply:
I saw you added versuon v0.0.1 above